sdk/monitor/Azure.Monitor.Query/MigrationGuideApplicationInsights.md

```diff
@@ -0,0 +1,163 @@
+
+### The client
+
+To provide a more intuitive experience, the top-level client to query logs was renamed to `LogsQueryClient` from `ApplicationInsightsDataClient`. `LogsQueryClient` can be authenticated using Azure AD. This client is the single entry point to execute a single Kusto query or a batch of Kusto queries.
+
+#### Consistency
+
+There are now methods with similar names, signatures, and locations to create senders and receivers. The result is consistency and predictability on the various features of the library.
+
+### Client constructors and authentication
+
+In `Microsoft.Azure.ApplicationInsights.Query` v1.0.0:
+
+```csharp
+using Microsoft.Azure.ApplicationInsights.Query;
+using Microsoft.Rest;
+using Microsoft.Rest.Azure.Authentication;
+
+// code omitted for brevity
+
+var adSettings = new ActiveDirectoryServiceSettings
+{
+    AuthenticationEndpoint = new Uri("https://login.microsoftonline.com"),
+    TokenAudience = new Uri("https://api.loganalytics.io/"),
+    ValidateAuthority = true
+};
+
+ServiceClientCredentials credentials = ApplicationTokenProvider.LoginSilentAsync(
+    "<domainId or tenantId>", "<clientId>", "<clientSecret>", adSettings).GetAwaiter().GetResult();
+var client = new ApplicationInsightsDataClient(credentials);
+```
+
+In `Azure.Monitor.Query` v1.0.x:
+
+```csharp
+using Azure.Monitor.Query;
+using Azure.Identity;
+
+// code omitted for brevity
+
+var credential = new ClientSecretCredential("<domainId or tenantId>", "<clientId>", "<clientSecret>");
+var client = new LogsQueryClient(credential);
+```
+
+### Send a single query request
+
+In `Microsoft.Azure.ApplicationInsights.Query` v1.0.0:
+
+```csharp
+using Microsoft.Azure.ApplicationInsights.Query.Models
+
+// code omitted for brevity
+QueryResults results = await client.Query.ExecuteAsync("<appId>", "AzureActivity | top 10 by TimeGenerated");
+```
+
+In `Azure.Monitor.Query` v1.0.x:
+
+- The `QueryBody` is flattened. Users are expected to pass the Kusto query directly to the API.
+- The `timespan` parameter is now required, which helps to avoid querying over the entire data set.
+
+```csharp
+using Azure;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+
+// code omitted for brevity
+
+Response<LogsQueryResult> response = await client.QueryWorkspaceAsync(
+	workspaceId,
+	"AzureActivity | top 10 by TimeGenerated",
+	new QueryTimeRange(TimeSpan.FromDays(1)));
+```
+
+### A note about the Events API
+
+The `Microsoft.Azure.ApplicationInsights.Query` package includes an Events API, which is just a different "API head" on the same logs data. It enables the querying of some logs in Application Insights without writing Kusto queries. The API translates to Kusto queries in the background. The same data can be accessed via regular Kusto queries, as shown in the preceding example.
+
+### Query metrics from a resource
+
+In `Microsoft.Azure.ApplicationInsights.Query` v1.0.0:
+
+```csharp
+using Microsoft.Azure.ApplicationInsights.Query.Models
```

Comment:
This example looks very different from what the Python migration guide shows in this section: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-query/migration_guide_app_insights.md#query-metrics-from-a-resource.

Shouldn't this call `client.Metrics.GetAsync(...)` instead of `client.QueryAsync(...)`?